tests/neg/t7319c.flags

```diff
@@ -0,0 +1 @@
+-Yno-import
```

Comment:
Dotty ignores flag files currently. 
You'll need to create a new test similarly to how it's done here:
https://github.com/lampepfl/dotty/blob/master/test/dotc/tests.scala#L128